res/openvpn-gui-res-it.rc

```diff
@@ -411,7 +411,8 @@ Per maggiori dettagli consulta il file log (%ls) ."
     IDS_NFO_SERVICE_STOPPED "Servizio OpenVPN arrestato."
     IDS_NFO_ACTIVE_CONN_EXIT "Ci sono ancora connessioni attive che verranno chiuse se esci dall'interfaccia di OpenVPN.\
 \n\nSei sicuro di voler uscire?"
-    IDS_ERR_PARSE_MGMT_OPTION "Impossibile analizzare l'opzione --management in <%ls%ls>."
+    IDS_ERR_PARSE_MGMT_OPTION "Impossibile analizzare l'opzione --management in <%ls%ls>. \
+L'allegato alla connessione automatica richiede l'opzione --management nel file config."
```

Comment:
Hi, was this the same sentence that was used before?
The English text says "Attaching to auto-started connections require --management option in the config file."
There is truly no "allegato" in here, please.

Reply:
Why is available in english language?

Reply:
"Could be ok "Il collegamento a connessioni avviate automaticamente richiede l'opzione --management nel file di configurazione."?

Reply:
Ok. Fixed. Thanks.

Reply:
please squash everything in one commit